emacs/merlin-ac.el

```diff
@@ -153,7 +161,6 @@ wrong then recompute it."
     (add-to-list 'ac-sources 'merlin-ac-source)))
 
 (add-hook 'merlin-mode-hook #'merlin-ac--setup)
-(when merlin-mode (merlin-ac--setup))
```

Comment:
Non-expert here: Is that call "non-sensical" because the function will and should be called via the hook `(add-hook 'merlin-mode-hook #'merlin-ac--setup)` ?